src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -1506,6 +1506,9 @@ public static void Main()
 ";
             var comp = CreateCompilation(src);
             comp.VerifyDiagnostics(
+                // (5,25): warning CS0067: The event 'C.X' is never used
+                //     public event Action X;
+                Diagnostic(ErrorCode.WRN_UnreferencedEvent, "X").WithArguments("C.X").WithLocation(5, 25)
```

Comment:
What part of the change caused us to start firing these diagnostics?

Reply:
IIRC these diagnostics started appearing once I deleted the UninitializedFieldsWalker--for much of my work, I simply commented out the part of that walker that gives the CS8618 diagnostic.